src/main/scala/togyzkumalak/variant/Bestemshe.scala

```diff
@@ -0,0 +1,47 @@
+package strategygames.togyzkumalak
+package variant
+
+import strategygames.togyzkumalak._
+import strategygames.{ GameFamily, Player }
+
+case object Bestemshe
+    extends Variant(
+      id = 2,
+      key = "bestemshe",
+      name = "Bestemshe",
+      standardInitialPosition = false,
+      boardSize = Board.Dim5x2
+    ) {
+
+  def gameFamily: GameFamily = GameFamily.Togyzkumalak()
+
+  def perfIcon: Char = ''
+  def perfId: Int    = 401
+
+  override def baseVariant: Boolean = false
+
+  override def canOfferDraw = false
+
+  // cache this rather than checking with the API everytime
+  override def initialFen =
+    format.FEN("5S,5S,5S,5S,5S/5S,5S,5S,5S,5S 0 0 S 1")
+
+  override def usesTuzdik = false
+
+  // TODO check legalMoves.size == 0 condition
+  override def specialEnd(situation: Situation) =
+    (situation.board.history.score.p1 > 25) ||
+      (situation.board.history.score.p2 > 25) ||
+      (situation.moves.size == 0)
+
+  override def specialDraw(situation: Situation) =
```

Comment:
I figured best to leave this clause in (which wont be triggered with regular Bestemshe) but when we get to the point where we are offering FromPosition for all variants then it could be possible to create a position (fen) where the scores start as odd and so a draw is possible. if i remove this code then i think it will just look in `winner` and award p2 as a winner in these 'tie' situations.